backend-rust/src/migrations/m0028_reindex_reward_metrics.sql

```diff
@@ -0,0 +1,6 @@
+ALTER TABLE metrics_rewards
```

Comment:
Please document the reason for the change, otherwise we might revert it in the future without understanding why it was here in the first place.